dashboard/config/locales/blocks.mr-IN.yml

```diff
@@ -842,7 +842,7 @@ mr:
             '"follow"': अनुसरण
             '"avoid"': टाळणे
       gamelab_allSpritesWithAnimation:
-        text: "[ANIMATION]"
+        text: "[ANIMATION][0]"
```

Comment:
I think it should be `{ANIMATION}` per https://github.com/code-dot-org/code-dot-org/blob/3ce15589ad3b5ea5acb7b49ae017202ef3b187a4/dashboard/config/blocks/GamelabJr/gamelab_allSpritesWithAnimation.json#L8. This file shouldn't contain redacted values.

Reply:
I agree with you Ha it looks like we should be using curly braces here.